srcpkgs/i3wsr/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'i3wsr'
+pkgname=i3wsr
+version=1.2.2
+revision=1
+build_style=cargo
+hostmakedepends="python3-devel"
```

Comment:
-devel files should go in makedepends, any particular reason it is in hostmakedepends?

Reply:
build needs python3 which won't work for cross when just in makedepends.

Reply:
Just tried it without and yes you are correct, but also just set it to python3 and it went fine, so just try with python3 in hostmake instead.

Reply:
done